app/models/concerns/sufia/ability.rb

```diff
@@ -52,6 +52,7 @@ def editor_abilities
 
       can :create, TinymceAsset
       can [:create, :update], ContentBlock
+      can :edit, ::SolrDocument
```

Comment:
I'm still unsure why we need this at all.  If we want the object to be editable by admins, shouldn't we add the admin group to the object's rights metadata?